lib/gmail-adapter.js

```diff
@@ -80,6 +89,10 @@ var Read = Juttle.proc.base.extend({
         // Build a search expression given the options.
         var search = self.raw || "";
 
+        if (this.filter_search_expr !== '') {
```

Comment:
Nit: If you are going to do this check, I'd initialize `this.filter_search_expr` to `null` (or not at all). On the otherhand, if its initialized to an empty string, it seems like you can concat it into `search` regardless of whether theres a filter or not?